backend/zeno_backend/server.py

```diff
@@ -260,15 +265,30 @@ def get_filtered_table(req: TableRequest, project_uuid: str, request: Request):
             )
         return filt_df.to_json(orient="records")
 
-    @api_app.post(
-        "/chart-data/{project}",
+    @api_app.get(
+        "/chart/{owner}/{project}/{chart_uuid}",
+        response_model=ChartResponse,
         tags=["zeno"],
-        response_model=str,
     )
-    def get_chart_data(chart: Chart, project: str, request: Request):
-        if not util.access_valid(project, request):
+    def get_chart(owner_name: str, project_name: str, chart_id: int, request: Request):
+        project_uuid = select.project_uuid(owner_name, project_name)
```

Comment:
Same as above.

Reply:
We don't necessarily have it. We could wait for the parent layout to resolve, but then it causes a waterfall and is slow.